src/app/containers/ImageWithPlaceholder/index.jsx

```diff
@@ -9,6 +10,10 @@ import { RequestContext } from '#contexts/RequestContext';
 
 const LAZYLOAD_OFFSET = 250; // amount of pixels below the viewport to begin loading the image
 
+const StyledImage = styled(Image)`
+  height: auto;
```

Comment:
is this not the default `height`? if so then I'm not sure it needs set again

Reply:
It's a bit annoying but for the `CLS` hack to work we still need to explicitly specify the `height` attribute both in the `<img />' tag and in CSS. This article attempts to explain why in detail: https://www.smashingmagazine.com/2020/03/setting-height-width-images-important-again/